workflow/tests/ACCA_Examples/Bob_Ross_Residence.xml

```diff
@@ -0,0 +1,1483 @@
+          </Floor>
+          <Floor>
+            <SystemIdentifier id='CeilingBathroom1'/>
+            <AttachedToSpace idref='bathroom_1'/>
+            <ExteriorAdjacentTo>attic - vented</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <FloorType>
+              <WoodFrame/>
+            </FloorType>
+            <Area>144.0</Area>
+            <Insulation>
+              <SystemIdentifier id='CeilingBathroom1Insulation'/>
+              <AssemblyEffectiveRValue>31.25</AssemblyEffectiveRValue>
+            </Insulation>
+          </Floor>
+          <Floor>
+            <SystemIdentifier id='CeilingFamilyRoom'/>
+            <AttachedToSpace idref='family_room'/>
+            <ExteriorAdjacentTo>attic - vented</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <FloorType>
+              <WoodFrame/>
+            </FloorType>
+            <Area>784.0</Area>
+            <Insulation>
+              <SystemIdentifier id='CeilingFamilyRoomInsulation'/>
+              <AssemblyEffectiveRValue>31.25</AssemblyEffectiveRValue>
+            </Insulation>
+          </Floor>
+          <Floor>
+            <SystemIdentifier id='CeilingKitchen'/>
+            <AttachedToSpace idref='kitchen'/>
+            <ExteriorAdjacentTo>attic - vented</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <FloorType>
+              <WoodFrame/>
+            </FloorType>
+            <Area>252.0</Area>
+            <Insulation>
+              <SystemIdentifier id='CeilingKitchenInsulation'/>
+              <AssemblyEffectiveRValue>31.25</AssemblyEffectiveRValue>
+            </Insulation>
+          </Floor>
+          <Floor>
+            <SystemIdentifier id='CeilingFoyer'/>
+            <AttachedToSpace idref='foyer'/>
+            <ExteriorAdjacentTo>attic - vented</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <FloorType>
+              <WoodFrame/>
+            </FloorType>
+            <Area>264.0</Area>
+            <Insulation>
+              <SystemIdentifier id='CeilingFoyerInsulation'/>
+              <AssemblyEffectiveRValue>31.25</AssemblyEffectiveRValue>
+            </Insulation>
+          </Floor>
+        </Floors>
+        <Slabs>
+          <Slab>
+            <SystemIdentifier id='SlabBedroom2'/>
+            <AttachedToSpace idref='bedroom_2'/>
+            <InteriorAdjacentTo>basement - conditioned</InteriorAdjacentTo>
+            <Area>304.0</Area>
+            <ExposedPerimeter>35.0</ExposedPerimeter>
+            <PerimeterInsulation>
+              <SystemIdentifier id='SlabBedroom2PerimeterInsulation'/>
+              <Layer>
+                <NominalRValue>5.0</NominalRValue>
+                <InsulationDepth>3.0</InsulationDepth>
+              </Layer>
+            </PerimeterInsulation>
```

Comment:
I realize the floor plan says R-5 edge insulation, but Table 7 doesn't even offer any options to describe a basement floor with edge insulation. I'm tempted to say that we should specify the basement slabs as uninsulated.

Reply:
Change it to be uninsulated for now.